eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -132,6 +132,31 @@ jobs:
   - checkout: self
     submodules: recursive
 
+  # Validate all the files are formatted correctly according to the .clang-format file.
+  - bash: |
+      # Run clang-format recursively on each source and header file within the repo sdk folder.
+      echo "Check clang-formatting (will use version 10)"
+      sudo apt-get update > /dev/null
+      sudo apt-get install clang-format-10 > /dev/null
```

Comment:
Why redirect to `/dev/null`? What motivated this? Does that hide some detail from the console output? Is it necessary?

Reply:
Can we consider downloading binary directly from https://releases.llvm.org/download.html#11.0.0 to sustain the conclusion we got in today's meeting?

Reply:
> Why redirect to /dev/null? 

I don't want to show the output of those commands as part of the log. It is irrelevant

Reply:
> Can we consider downloading binary directly 

Will try it. Makes me a little worry to think if the link could ever go down and break the build. But just a little :P

Reply:
> I don't want to show the output of those commands as part of the log. It is irrelevant

Won't it be useful to know the reason if the CI step was to fail due to `sudo apt-get install clang-format-10` (for example if it was because of networking)? Is it a lot of logs that get printed? I don't see any harm in keeping it.